sentences/fr/homeassistant_HassSetPosition.yaml

```diff
@@ -0,0 +1,10 @@
+language: fr
+intents:
+  HassSetPosition:
+    data:
+      - sentences:
```

Comment:
We're missing commun sentences with `<ouvre>` and `<ferme>`
It's very commun to say "ouvre le volet a 80%" or "Ferme le volet a 80%"

I know some platforms are doing some super smart things when it comes to Open to X% vs Close to X% but I would propose to make it dead simple:
Both opening or closing to X% should **set the entity to X%** and that's it

- Ouvre le volet a 80%: Set the cover to 80%
- Ferme le voelt a 80%: Set the cover to 80%

They are similar but won't be used in the same condition, most of the time we say "Open to X%" when the current position is lower and "Close to X%" when it's higher.

A conversation may look like this
- *Cover closed*
- Open the cover to 70%
- *Cover at 70%*
- *Wow that's too much light*
- Close the cover to 60%
- *Cover at 60%*
- Perfect